created a brand handbook, aka brandbook (#16855)
PostHog/posthog.com: contents/handbook/brand/in-practice.md

Social media channels can be more casual and reactive than the website, but they should never feel like a corporate account posting congratulatory content or generic "tips."

## Marketing emails

Same voice as the website, but can be [slightly more casual](/handbook/marketing).

## Presentations

PostHog uses Figma Slides for polished presentations. See the [communication guidelines](/handbook/company/communication) for more.

### When designing a deck

- Fewer words per slide. One idea per slide.
- Don't use slide templates that look like every other B2B SaaS deck.
- Use the templates supplied by the Graphics Team. Use of the Squeak font is fine for punchy visual moments.
- PostHog color palette throughout.
- Hedgehogs can appear – use the art library.

### Informal decks (internal, demos)

These have more flexibility. Speed > perfection. Use the existing templates and don't spend hours on formatting.

Our board meeting deck still uses a title slide with branding circa 2021, but it (sort of) intentionally shows we put our focus in the right places, not in making pretty decks for people who already believe in us.

## Events & conference booths

> Events are a high-density brand moment where first impressions form in seconds. Booth design, event-specific assets, and staffing guidance live in the [marketing/events handbook](/handbook/marketing/events).

Reviewer: ```suggestion
Events are a high-density brand moment where first impressions form in seconds. Booth design, event-specific assets, and staffing guidance live in the [Marketing events handbook](/handbook/marketing/events).
```

## Print materials

Print requires additional consideration because you can't iterate after it's shipped.

### General rules for print

- Export at 300 DPI minimum for anything that will be physically produced.

> - CMYK color profile for print. (Note: our brand colors are defined in RGB/hex. Confirm CMYK equivalents with the design team before going to print.) @todo add Pantone colors here.

Reviewer: ```suggestion
- CMYK color profile for print. (Note: our brand colors are defined in RGB/hex. Confirm CMYK equivalents with the design team before going to print.)
```

- Bleed and safe zones: Use standard 3mm bleed. Keep critical content at least 5mm inside the trim line.
- SVG logos always preferred as source; export as PDF for print vendor.

### Materials that commonly need to be produced

- Posters: Think billboard-first. One message. One big visual.
- Brochures: Rare. If needed, follow the website's editorial style and make it look handcrafted, not template-y.
- We've traditionally avoided sell sheets or business cards, but work with the Graphics Team if you need something specific.

> ### Approval:** All print materials should be reviewed by in #design-review before going to print. Mistakes in print are expensive. [Submit a request](/handbook/brand/art-requests) early.

Reviewer: ```suggestion
### Approval:

All print materials should be reviewed by in `#design-review` before going to print. Mistakes in print are expensive. [Submit a request](/handbook/brand/art-requests) early.
```

> ### Approval:** All print materials should be reviewed by in #design-review before goito print. Mistakes in print are expensive. [Submit a request](/handbook/brand/art-requests) early.

Reviewer 2: ```suggestion
### Approval:** All print materials should be reviewed by in #design-review before going to print. Mistakes in print are expensive. [Submit a request](/handbook/brand/art-requests) early.
```
